test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -1293,14 +1293,20 @@ private static synchronized void showUI() {
 
 
     /**
-     * Forcibly pass the test.
+     * Forcibly pass the test. This should be used in semi-automatic tests when
```

Comment:
```suggestion
     * Forcibly pass the test. This method should be used in semi-automatic tests when
```
I'm inclined to use imperative: _“Use this method in….”_